src/main/java/com/board/config/boardconfiggui/HelloApplication.java

```diff
@@ -18,6 +25,24 @@ public void start(Stage stage) throws IOException {
     }
 
     public static void main(String[] args) {
+        initialize();
         launch();
     }
+
+    private static void initialize() {
```

Comment:
Create a separate thread/service for initialization and invoke the launch method after completion of initialization (Use Platform.runLater())